src/d4pkg/d4.nim

```diff
@@ -241,3 +244,6 @@ when isMainModule:
   var o = d4f.values("chr2", 0, 60)
   echo o
   doAssert o[12] == 2'i32
+  doAssert d4.index_build_sfi("test.d4")
```

Comment:
is there a way to test this has been created?

Reply:
Yeah I was thinking that and started working on it, but the  index type and check index function in the c api only supported the sum index, not sfi. Though now thinking about it again, i could just add a check sfi function. 

Other way to check which I've been using is to just call d4tools framedump and make sure the .index is there

Reply:
ok. no need.